src/libraries/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -863,7 +863,7 @@ public sealed partial class DataTableReader : System.Data.Common.DbDataReader
         public override System.Type GetProviderSpecificFieldType(int ordinal) { throw null; }
         public override object GetProviderSpecificValue(int ordinal) { throw null; }
         public override int GetProviderSpecificValues(object[] values) { throw null; }
-        public override System.Data.DataTable? GetSchemaTable() { throw null; }
+        public override System.Data.DataTable GetSchemaTable() { throw null; }
```

Comment:
This looks wrong:
https://github.com/dotnet/runtime/pull/41082
cc: @roji

Reply:
I'll revert this

Reply:
Sorry for the delayed response and thanks - yeah, this is wrong. GetSchemaTable should return nullable.

Reply:
This was changed back per https://github.com/dotnet/runtime/pull/41731#discussion_r485268845